pkgs/os-specific/linux/sgx/azure-dcap-client/default.nix

```diff
@@ -79,6 +80,10 @@ stdenv.mkDerivation rec {
     "DESTDIR=$(out)"
   ];
 
+  # Online test suite; run with
+  # $(nix-build -A sgx-azure-dcap-client.tests.suite)/bin/tests
+  passthru.tests.suite = callPackage ./test-suite.nix { };
```

Comment:
Is this a common pattern in `nixpkgs`?

Reply:
We have something similar for the SGX hardware tests: 

https://github.com/NixOS/nixpkgs/blob/e76c78d20685a043d23f5f9e0ccd2203997f1fb1/pkgs/os-specific/linux/sgx/sdk/default.nix#L260-L276

I think it makes sense to have the test suite available to easily test upgrades.

Reply:
Yeah, we can do that to move the tests out of the normal package build.